14-foundations-mathematical.Rmd

```diff
@@ -1172,7 +1172,7 @@ The conditions for applying the normal model have already been verified, so we c
 $$\text{point estimate} \pm 1.96 \times SE = 0.090 \pm 1.96 \times 0.028 = (0.035, 0.145)$$
 
 We are 95% confident that implanting a stent in a stroke patient's brain increased the risk of stroke within 30 days by a rate of 0.035 to 0.145.
-This confidence interval can also be used in a way analogous to a hypothesis test: since the interval does not contain 0 (is completely above 0), it means the data provide statistically significant evidence that the stent used in the study *increases* the risk of stroke within 30 days.
+This confidence interval can also be used in a way analogous to a hypothesis test: since the interval does not contain 0 (is completely above 0), it means the data provide strong evidence that the stent used in the study *increases* the risk of stroke within 30 days.
```

Comment:
```suggestion
This confidence interval can also be used in a way analogous to a hypothesis test: since the interval does not contain 0 (is completely above 0), it means the data provide convincing evidence that the stent used in the study *increases* the risk of stroke within 30 days.
```